pkg/webhook/deployment.go

```diff
@@ -47,6 +49,17 @@ func CreateWebhookServerDeployment(
 			return err
 		}
 		deployment.ResourceVersion = existingCfg.ResourceVersion
+
+		//Add restart timestamp which will update the webhook server
+		//deployment to force restart. This is done so that the
+		//serviceaccount userid is updated to use the latest and the
+		//web-terminal does not hang.
+		if deployment.Spec.Template.Annotations == nil {
+			deployment.Spec.Template.Annotations = map[string]string{}
+		}
+		now := time.Now()
+		deployment.Spec.Template.Annotations[constants.RestartAtAnnotation] = strconv.FormatInt(now.UnixNano(), 10)
```

Comment:
what if we move it into getSpecDeployment?

Reply:
Ah, I see. Then it will be assigned to newly created resource... 

I don't have a strong opinion.
A bit better to make code a bit simpler

Reply:
Ohhh okay, i see, it works that way as well and it is cleaner, sorry, I wasn't sure if that would work so i put it above the update. I've changed to code to add it to the getSpecDeployment func though! Thanks for the suggestion @sleshchenko!